miner/remote_proposals.go

```diff
@@ -195,6 +203,7 @@ func (pb *RemoteProposalBuilder) build(
 				panic("missing node epoch eligibilities")
 			}
 		}
+		pb.identityStates.SetEligibilities(nodeId, proofs)
```

Comment:
The eligibilities shouldn't change mid-epoch. They can be set once, when calculated in:
```go
proofs = calcEligibilityProofs(
	signer.signer.VRFSigner(),
	epoch,
	bcn,
	types.VRFPostIndex(nonce),
	proposal.Ballot.EpochData.EligibilityCount,
	pb.cfg.layersPerEpoch,
)
eligibilities[nodeId] = proofs
pb.identityStates.SetEligibilities(nodeId, proofs)
```